apps/browser/qml/pages/PrivacySettingsPage.qml

```diff
@@ -118,6 +189,13 @@ Page {
         }
     }
 
+    ConfigurationValue {
+        id: clearHistoryPeriodConfig
+
+        key: "/apps/sailfish-browser/actions/clear_history_period"
```

Comment:
Same as https://github.com/sailfishos/sailfish-browser/pull/912#discussion_r717437831 CC @atatarov

Reply:
PR https://github.com/sailfishos/sailfish-browser/pull/912 merged, please rebase and update this accordingly.